src/components/Navbar/MobileNav.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import { IconBrandGithub } from '@tabler/icons-react';
+import { useLocation } from 'react-router-dom';
+import { Burger } from '@mantine/core';
+import logo from '../../../public/logo.svg';
+import { Button } from '../UI/Button/Button';
+import { Divider } from '../UI/Divider/Divider';
+import { Link } from '../UI/Link/Link';
+import { Typography } from '../UI/Typography/Typography';
+import { NAVIGATION_LINKS } from './Navbar';
+import { mobileStyles as styles } from './styles';
+
+export default function MobileNav() {
+  const [displayLinks, setDisplayLinks] = useState(false);
+  const location = useLocation();
+  const pathname = location.pathname;
+
+  function handleDisplayLinks() {
+    setDisplayLinks((prev) => !prev);
+  }
+
+  return (
+    <div>
+      <div style={styles.container}>
+        <div style={styles.headerContainer}>
+          <img alt='Deadlink logo' style={styles.logoSize} src={logo} />
+          <Link labelStyle={styles.header} href='/' label='Deadlink-Hunter' disableHover />
+        </div>
+        <Burger
+          style={styles.burger}
+          size={styles.burgerSize}
+          opened={displayLinks}
+          onClick={handleDisplayLinks}
+        />
+      </div>
+      {displayLinks && (
+        <div style={styles.linksContainer}>
+          {NAVIGATION_LINKS.map((link) => {
+            const isPathMatch = pathname === link.href;
+            return (
+              <Link

```

Comment:
I think that its better to use the react router dom component for this (Wrote a comment about it), and this way you may not need to have all of the styles for the "is current selected" and use something built it